scripts/parallel-script/README.md

```diff
@@ -32,10 +32,15 @@ These scripts will help you to start a multi thread processment either for `aws
         file2
         file3
         ```
+        💡 You can use the AWS CLI to create your list of files, here is how to do it:
+
+        ```
+        aws s3 ls s3://ORIGIN_BUCKET --recursive | awk '{$1=$2=$3=""; print $0}' | sed 's/^[ \t]*//' > my_files_list.txt & tail -F my_files_list.txt
```

Comment:
if you want to display each file size and the total of files you could also use https://github.com/DNXLabs/tools-box/blob/master/scripts/s3-migration/sync.sh#L10